gammapy/scripts/image_pipe.py

```diff
@@ -11,242 +11,100 @@
 from ..image import SkyMap, SkyMapCollection, disk_correlate
 from ..stats import significance
 
-__all__ = ['ImageAnalysis']
+__all__ = ['ObsImage',
+           'MosaicImage']
 
 log = logging.getLogger(__name__)
 
 
-class ImageAnalysis(object):
-    """Gammapy 2D image based analysis.
+class ObsImage(object):
+    """Gammapy 2D image based analysis for one observation.
 
     The computed images are stored in a ``maps`` attribute of type `~gammapy.image.SkyMapCollection`
     with the following keys:
 
     * counts : counts for one obs
     * bkg : bkg model for one obs
     * exposure : exposure for one obs
-    * total_counts : total counts of all the observations
-    * total_bkg : total bkg model for all the observations
-    * total_exposure : total exposure for all the observations
-    * total_excess : total excess for all the observations
-    * significance : significance for all the observations
+    * excess : excess for one obs
+    * significance : significance for one obs
 
     Parameters
     ----------
+    events : `~gammapy.data.EventList`
+        Event Table of the observation
+    data_store : `~gammapy.data.DataStore`
+        `DataStore` where are situated the events
     empty_image : `~gammapy.image.SkyMap`
             ref to an empty image
     energy_band : `~gammapy.utils.energy.Energy`
         Energy band for which we want to compute the image
     offset_band : `astropy.coordinates.Angle`
         Offset Band where you compute the image
-    data_store : `~gammapy.data.DataStore`
-        `DataStore` where are situated the events
-    obs_table : `~astropy.table.Table`
-            Required columns: OBS_ID
     exclusion_mask : `~gammapy.image.ExclusionMask`
             Exclusion regions
     ncounts_min : int
-            Minimum counts required for one observation
+            Minimum counts required for the observation
     """
 
-    def __init__(self, empty_image=None,
-                 energy_band=None, offset_band=None,
-                 data_store=None, obs_table=None, exclusion_mask=None, ncounts_min=0):
-        self.maps = SkyMapCollection()
-
-        self.data_store = data_store
-
-        self.obs_table = obs_table
+    def __init__(self, events, data_store, empty_image,
```

Comment:
Minor suggestion: can you slightly change the class, so that it takes a `DataStoreObservation` object instead of the event list and the data store? I think this makes in the end a clearer API:

```
obs = data_store.obs(obs_id)
image_analyis = ObsImage(obs, ...)
```